src/stories/components/CioPlp/CioPlpProps.md

```diff
@@ -1,3 +1,41 @@
+### `useShopifyDefaults`
+
+---
+
+When enabled, this prop applies Shopify specific default configurations for callbacks and URL helpers. Custom callbacks and URL helpers can still override these defaults.
```

Comment:
Let's not specify which configurations. In the near future, we may want to also have itemFieldGetter defaults